src/Goerli-DssSpell.t.sol

```diff
@@ -484,51 +539,332 @@ contract DssSpellTest is GoerliDssSpellTestBase {
+
+        assertEq(DSValueAbstract(pip).read(), bytes32(260 * MILLION * WAD), "RWA007: Bad PIP value after bump()");
+    }
+
+    function testRWA007_INTEGRATION_TELL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        (, , , uint48 tocPre) = oracle.ilks("RWA007-A");
+        assertEq(uint256(tocPre), 0, "RWA007: `toc` is not 0 before tell()");
+        assertTrue(oracle.good("RWA007-A"), "RWA007: Oracle not good before tell()");
+
+        vat.file("RWA007-A", "line", 0);
+        oracle.tell("RWA007-A");
+
+        (, , , uint48 tocPost) = oracle.ilks("RWA007-A");
+        assertGt(uint256(tocPost), 0, "RWA007: `toc` is not set after tell()");
+        assertTrue(!oracle.good("RWA007-A"), "RWA007: Oracle still good after tell()");
+    }
+
+    function testRWA007_INTEGRATION_TELL_CURE_GOOD() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        vat.file("RWA007-A", "line", 0);
+        oracle.tell("RWA007-A");
+
+        assertTrue(!oracle.good("RWA007-A"), "RWA007: Oracle still good after tell()");
+
+        oracle.cure("RWA007-A");
+
+        assertTrue(oracle.good("RWA007-A"), "RWA007: Oracle not good after cure()");
+        (, , , uint48 toc) = oracle.ilks("RWA007-A");
+        assertEq(uint256(toc), 0, "RWA007: `toc` not zero after cure()");
+    }
+
+    function testFailRWA007_INTEGRATION_CURE_BEFORE_TELL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(oracle), address(this));
+
+        oracle.cure("RWA007-A");
+    }
+
+    function testRWA007_INTEGRATION_TELL_CULL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        assertTrue(oracle.good("RWA007-A"));
+
+        vat.file("RWA007-A", "line", 0);
+        oracle.tell("RWA007-A");
+
+        assertTrue(!oracle.good("RWA007-A"), "RWA007: Oracle still good after tell()");
+
+        oracle.cull("RWA007-A", addr.addr("RWA007_A_URN"));
+
+        assertTrue(!oracle.good("RWA007-A"), "RWA007: Oracle still good after cull()");
+        (, address pip, , ) = oracle.ilks("RWA007-A");
+        assertEq(DSValueAbstract(pip).read(), bytes32(0), "RWA007: Oracle PIP value not set to zero after cull()");
+    }
+
+    function testRWA007_PAUSE_PROXY_OWNS_RWA007_TOKEN_BEFORE_SPELL() public {
+        assertEq(rwagem_007.balanceOf(addr.addr('MCD_PAUSE_PROXY')), 1 * WAD);
+    }
+
+    function testRWA007_SPELL_LOCK_OPERATOR_DRAW_WIPE_FREE() public {
```

Comment:
note: most of the specific tests could be merged into a `checkRwaIlkIntegration` in base spell tests, generalising where possible to ease review, following base integration test patterns (to be done in a separate pr eventually, for future onboardings).